src/operations/course.ts

```diff
@@ -0,0 +1,60 @@
+import { get } from 'http';
+import getDb from '../initDB';
+import { Course, CourseCreate, CourseUpdate,} from '../models';
+
+const db = getDb();
+
+export const createCourse = async (course: CourseCreate): Promise<Course> => {
+  return db.course.create({ data: course });
+};
+
+export const getCourseById = async (
+  id: number,
+  ownerId: string
+): Promise<Course> => {
+  try {
+    const course = await db.course.findFirst({ where: { id, ownerId } });
+    if (!course) {
+      throw new Error('Course not found');
+    }
```

Comment:
could this be removed if we were to instead use `findFirstOrThrow`? it sounds like then an error would be automatically thrown if the course isn't found. but this totally works!

Reply:
I think what Isaiah says here about `findFirstOrThrow` might be a good choice since that is all we are doing with our check anyways